UI/View.js

```diff
@@ -0,0 +1,168 @@
+                </button>
+            </div>
+            <div class = "description">
+                <span>${post._descriprion}</span>
+                <div class = "tags">
+                    ${post._hashtags.join(',')}
+                </div>
+            </div>
+          </div>
+        
+    `;
+
+    posts.appendChild(postEl);
+  }
+
+  _showPost(post) {
+    let photos = document.getElementById('photoPosts');
+    this._addViewPost(post, photos);
+  }
+  showPosts(posts) {
+    let photos = document.getElementById('photoPosts');
+    console.log("Output \n");
+    posts.forEach((v1) => {
+      console.log(v1);
+      this._addViewPost(v1, photos);
+    });
+
+  }
+
+  clearPosts() {
+    let posts = document.getElementById("photoPosts");
+    posts.innerHTML = ``;
+    
+  }
+  _removePost(id = 0) {
+    let post = document.getElementById(id);
+    if (post) {
+      post.remove();
+    }
+  }
+
+  _editPost(id, newPost) {
+    let oldPost = document.getElementById(id);
+    if (oldPost) {
+      oldPost.firstElementChild.setAttribute('src', newPost._photoLink);
+      oldPost.querySelector('.description span').textContent = newPost._descriprion;
+      oldPost.querySelector('.description .tags').textContent = newPost._hashtags.join(',');
+    }
+  }
+
+  doLike(id, flag){
+
+    let post = document.getElementById(id);
+    let like = post.querySelector('#like'); 
+    like.firstElementChild.setAttribute('src','img/like.png');
+    like.style.background = "hsla(307, 29%, 56%, 0.7)";
+    if(flag){      
+      let like = post.querySelector('#like');
+      like.firstElementChild.setAttribute('src','img/like_red.png');
+      like.firstElementChild.style.background = "hsla(307, 29%, 56%, 0.7)";
+      
+    }
+    
+  }
+
+  setAuthorized(user) {
+    let headerAct = document.getElementById('username');
+    let userName = user && user.getUserName();
+    headerAct.textContent = userName || 'Anonim';
+    document.body.classList.toggle('is-auth', user.isUserName());
+  }
+
+  login(){
+    const log = document.getElementById('login');
+    log.innerHTML = `
+    <label >
+        <div  class = "font"> Login </div>
+        <input type="text" id ="user-login" class = "inputs" placeholder = "Input login" autocomplete="off" />
+    </label>
+            
+    <label>
+        <div class="font">Password</div>
+        <input type="text" id="user-password" class = "inputs" placeholder = "Input Password" autocomplete="off"/>
+    </label>
+    `;
+  }
+
+  error(){
+    const err = document.getElementById('error');
+    err.innerHTML = `
+    <div class = "font error">
+      <img  src="img/err.jpg" class = "error" alt = 'Invalid photo'>
+    </div>
+    <h3 class = "font error">Input Error</h3>
+    <h3 class = "font error">Please, try again...</h3> 
+    `;
+  }
+
+  pageAddPost(user){
+    const post = document.getElementById('btn-add');
+    post.innerHTML = `
+    <div class = "font">Your photoPost</div>
```

Comment:
Why can't we just have it in the markup inside the dialog element ?
I know you like dificulties but do it easely Julia ?